samples/app-anonymous-users/csharp/TeamsAppManifest/manifest.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.teams.sample.appAnonymousUsers",
+  "developer": {
+    "name": "appAnonymousUsers",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "name": {
+    "short": "Anonymous User Support",
+    "full": "Anonymous User Support"
+  },
+  "description": {
+    "short": "This sample shows Anonymous Users Support In Meeting.",
+    "full": "This sample shows Anonymous Users Support In Meeting."
+  },
+  "accentColor": "#FFFFFF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://{{domain-name}}/configure",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "team",
+        "groupchat"
+      ],
+      "context": [
+        "meetingSidePanel",
+        "meetingStage"
+      ]
+    }
+  ],
+  "bots": [
+    {
+      "botId": "<<YOUR-MICROSOFT-APP-ID>>",
+      "scopes": [
+        "groupchat",
+        "team"
+      ],
+      "isNotificationOnly": false,
+      "supportsCalling": false,
+      "supportsVideo": false,
+      "supportsFiles": false,
+      "commandLists": [
+        {
+          "scopes": [
+            "groupchat",
+            "team"
+          ],
+          "commands": [
+            {
+              "title": "Vote",
+              "description": "Sends message with @mention of the sender"
+            },
+            {
+              "title": "CreateConversation",
+              "description": "One-on-one conversation with an anonymous user"
```

Comment:
Create 1:1 conversation with bot for all the meeting participants via @mention CreateConversation

Reply:
Fixed